src/types.js

```diff
@@ -14,7 +14,7 @@ export type PromiseType = {
   parliament: ParliamentType,
   created: string,
   modified: string,
-  politician: PoliticianIdType,
+  politician: *,
```

Comment:
[Existential type](https://flow.org/en/docs/types/utilities/#toc-the-existential-type)

Flow will try to infer the type here.
Will manually type this once state is normalized